### PR TITLE
ICALL_EXPORT reduction.

### PR DESCRIPTION
**Pull request by jaykrell**
ICALL_EXPORT should not be placed on HANDLES() functions.
Only the wrappers get it.
And the declarations are generally redundant.

### CODE REVIEW

**Comment by jaykrell** [comment]
@monojenkins build Linux AArch64 Interpreter

**Comment by jaykrell** [comment]
@monojenkins build Linux AArch64 Interpreter

File: mono/metadata/object-internals.h [hunk 1]
```diff
@@ -2118,47 +2092,16 @@ ICALL_EXPORT
...
-ves_icall_ModuleBuilder_GetRegisteredToken (MonoReflectionModuleBuilderHandle mb, guint32 token, MonoError *error);
-
 ICALL_EXPORT
 void
 ves_icall_AssemblyBuilder_basic_init (MonoReflectionAssemblyBuilder *assemblyb);
 
-ICALL_EXPORT
-void
-ves_icall_AssemblyBuilder_UpdateNativeCustomAttributes (MonoReflectionAssemblyBuilderHandle assemblyb, MonoError *error);
+//void ves_icall_AssemblyBuilder_UpdateNativeCustomAttributes (MonoReflectionAssemblyBuilderHandle assemblyb, MonoError *error);
```
Comment (lambdageek) [inline mono/metadata/object-internals.h]: Why is this one commented out and not deleted?

**Comment by jaykrell** [comment]
@monojenkins build failed

**Pull request merged**